buildpipeline/pipeline.json

```diff
@@ -141,8 +140,7 @@
           "Parameters": {
             "PB_AdditionalBuildArguments": "/p:PortableBuild=true /p:SkipTests=true /p:NativeToolSetDir=C:\\tools\\clr",
             "PB_TargetArchitecture": "arm64",
-            "PB_PortableBuild": "true",
-            "PB_DistroRid": "win-arm64"
+            "PB_PortableBuild": "true"
```

Comment:
Talked with @eerhardt , I think @wtgodbe has a signing change that needs to have separate steps for build and publish.